src/components/elements/table/EnhancedTableToolbar.tsx

```diff
@@ -42,9 +42,11 @@ const EnhancedTableToolbar = <T extends { id: string }>({
       }}
     >
       <Stack
-        justifyContent='space-between'
-        alignItems={'center'}
-        direction={isTiny ? 'column' : 'row'}
+        justifyContent={stackVertically ? undefined : 'space-between'}
```

Comment:
These are unrelated improvements to the EnhancedTableToolbar on mobile. Before (current QA)
<img width="320px" src="https://github.com/user-attachments/assets/12f58e3a-a834-4375-bed6-3ff364c53e4f">

After (local)
<img width="320px" src="https://github.com/user-attachments/assets/16e0340d-49a3-4bb0-859e-cb132f516e54">